### ISSUE TITLE
multiple display of same for comments  in esteem app

### ISSUE THREAD

**Issue by ridvanmb** [eSteemApp/esteem#203]
Today, I ran into a bug when I was wandering through eSteem. 
When I went to my profile everything seemed normal
Then from the Blog - Comments-Replies- Wallet sections below the page I clicked on my wallet.
After checking my wallet, I tried to open the "Comments" page and It didn't open, so I clicked 2-3 more times and it did open this time. But when it opened there were multiple copies of the same comment. I realized the comments were showing as many times as I clicked on the comments section. So when I clicked 3 times the same comment was displayed as 3 separate comments but of course, with the came content. 
I closed and relaunched the app, but the same problem continued on happening. ![1.png](https://res.cloudinary.com/hpiynhbhq/image/upload/v1512995188/vh0r1mrwmnchbapfgcvd.png)
I am using the latest version of eSteem. When I tried the same thing on my friend's phone, the same bug occurred on his phone too.
In conclusion, I found a bug in eSteem's "comments" section.

<br /><hr/><em>Posted on <a href="https://utopian.io/utopian-io/@rdvn/multiple-display-of-same-for-comments-in-esteem-app">Utopian.io -  Rewarding Open Source Contributors</a></em><hr/>
